src/character.h

```diff
@@ -402,14 +402,12 @@ class Character : public Creature, public visitable<Character>
         bool worn_with_flag( std::string flag ) const;
 
         // --------------- Skill Stuff ---------------
-        SkillLevel &skillLevel(const Skill* _skill);
-        SkillLevel &skillLevel(Skill const &_skill);
-        SkillLevel &skillLevel(const skill_id &ident);
+        SkillLevel &get_skill_level( const skill_id &ident );
```

Comment:
It's currently still used to set a few properties of the `SkillLevel` object, e.g. train, rust, exercise, etc.

I though about (later) making the function protected, or accessing `Character::_skills` directly.